.nuxt/App.js

```diff
@@ -0,0 +1,200 @@
+import Vue from 'vue'
```

Comment:
.nuxt directory shouldn't be push to template too.

Reply:
Yap, its removed